### PR TITLE
ARROW-17282: [Python] flake8 update fails linter CI

### PR DESCRIPTION
**Pull request by assignUser**

### CODE REVIEW

**Review by raulcd — COMMENTED** [review]
Thanks @assignUser!! Only one minor thing, can you also update this version to the latest one to avoid getting inconsistent results between pre-commit and the job?
https://github.com/apache/arrow/blob/master/.pre-commit-config.yaml#L40

**Review by raulcd — APPROVED** [review]

**Comment by raulcd** [comment]
cc~ @jorisvandenbossche

**Comment by raulcd** [comment]
cc~ @pitrou

File: dev/archery/archery/utils/command.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 
 def default_bin(name, default):
-    assert(default)
+    assert (default)
```
Comment (pitrou) [inline dev/archery/archery/utils/command.py:27]:
```suggestion
    assert default
```

File: python/pyarrow/tests/parquet/test_encryption.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def write_encrypted_parquet(path, table, encryption_config,
                             kms_connection_config, crypto_factory):
     file_encryption_properties = crypto_factory.file_encryption_properties(
         kms_connection_config, encryption_config)
-    assert(file_encryption_properties is not None)
+    assert (file_encryption_properties is not None)
```
Comment (pitrou) [inline python/pyarrow/tests/parquet/test_encryption.py:112]: No need for parentheses around assert statements. Can you make the change in other instances below as well?
```suggestion
    assert file_encryption_properties is not None
```

**Review by pitrou — APPROVED** [review]
+1, thanks for the fix @assignUser !

**Comment by pitrou** [comment]
@assignUser Can you perhaps enable Github Actions on your fork?

**Comment by assignUser** [comment]
@pitrou https://github.com/assignUser/arrow/actions/runs/2789744614

**Comment by pitrou** [comment]
@assignUser Hmm, pity this didn't actually run the lint job.

**Comment by pitrou** [comment]
Lint job here, can merge: https://github.com/pitrou/arrow/actions/runs/2790029916

**Pull request merged**